Model/Resolver/Invoice/InvoiceItems.php

```diff
@@ -162,7 +162,7 @@ protected function formatDiscountDetails(OrderInterface $associatedOrder, Invoic
             $discounts[] = [
                 'label' => $associatedOrder->getDiscountDescription(),
                 'amount' => [
-                    'value' => abs($invoiceItem->getDiscountAmount()) ?? 0,
+                    'value' => abs((int)$invoiceItem->getDiscountAmount()) ?? 0,
```

Comment:
I believe the type casting will convert null to 0, then we down need the coalesce operator here.
Please check it and if it's true, remove it, please.